example/transferAsset/main.go

```
@@ -0,0 +1,110 @@
+package main
+
+import (
+	"crypto/rand"
+	"encoding/hex"
+	"fmt"
+
+	"github.com/CodeChain-io/codechain-sdk-go/core"
+	"github.com/CodeChain-io/codechain-sdk-go/core/transaction"
+	"github.com/CodeChain-io/codechain-sdk-go/key"
+	"github.com/CodeChain-io/codechain-sdk-go/primitives"
+	"github.com/CodeChain-io/codechain-sdk-go/rpc"
+)
+
+func main() {
+
+	rpcClient := rpc.NewRPC("http://localhost:8080")
+	networkID := "tc"
+
+	signerAccountID, _ := primitives.PlatformAddressFromString("tccq9h7vnl68frvqapzv3tujrxtxtwqdnxw6yamrrgd")
+	signerSecret, _ := primitives.StringToH256("ede1d4ccb4ec9a8bbbae9a13db3f4a7b56ea04189be86ac3a6a439d9a0a1addd")
```

Let's not ignore errors. We should assert that these errors == nil.